chapters/data-analysis.tex

```diff
@@ -1,292 +1,488 @@
+We attribute some of this to the difficulty of writing code to create them.
+Making a visually compelling graph would already be hard enough if you didn't have to go through many rounds of googling to understand a command.
+The trickiest part of using plot commands is to get the data in the right format.
+This is why the \textbf{Stata Visual Library} includes example data sets to use 
+with each do-file.
+
+Whole books have been written on how to create good data visualizations,
+so we will not attempt to give you advice on it.
+Rather, here are a few resources we have found useful.
 The Tapestry conference focuses on ``storytelling with data''.\sidenote{
-\url{https://www.youtube.com/playlist?list=PLb0GkPPcZCVE9EAm9qhlg5eXMgLrrfMRq}}
+	\url{https://www.youtube.com/playlist?list=PLb0GkPPcZCVE9EAm9qhlg5eXMgLrrfMRq}}
 \textit{Fundamentals of Data Visualization} provides extensive details on practical application;\sidenote{
-\url{https://serialmentor.com/dataviz}}
+	\url{https://serialmentor.com/dataviz}}
 as does \textit{Data Visualization: A Practical Introduction}.\sidenote{
-\url{http://socvis.co}}
+	\url{http://socvis.co}}
+Graphics tools like Stata are highly customizable.
+There is a fair amount of learning curve associated with extremely-fine-grained adjustment,
+but it is well worth reviewing the graphics manual\sidenote{\url{https://www.stata.com/manuals/g.pdf}}
+For an easier way around it, Gray Kimbrough's \textit{Uncluttered Stata Graphs} code is an excellent default replacement for Stata graphics that is easy to install.
+\sidenote{\url{https://graykimbrough.github.io/uncluttered-stata-graphs/}}
+If you are a R user, the \textit{R Graphics Cookbook}\sidenote{\url{https://r-graphics.org/}} 
+is a great resource for the most popular visualization package \texttt{ggplot}\sidenote{\url{https://ggplot2.tidyverse.org/}}. 
+But there are a variety of other visualization packages, 
+such as \texttt{highcharter}\sidenote{\url{http://jkunst.com/highcharter/}}, 
+\texttt{r2d3}\sidenote{\url{https://rstudio.github.io/r2d3/}}, 
+\texttt{leaflet}\sidenote{\url{https://rstudio.github.io/leaflet/}}, 
+and \texttt{plotly}\sidenote{\url{https://plot.ly/r/}}, to name a few.
+We have no intention of creating an exhaustive list, and this one is certainly missing very good references.
+But at least it is a place to start.
+
+\section{Exporting analysis outputs}
+
+% Exploratory analysis
+It's ok to not export each and every table and graph created during exploratory analysis. 
+Instead, we suggest running them into markdown files using RMarkdown or the different dynamic document options available in Stata. 
+This will allow you to update and present results quickly while maintaining a record of the different analysis tried. 
+% Final analysis
+Final analysis scripts, on the other hand, should export final outputs, which are ready to be included to a paper or report.
+No manual edits, including formatting, should be necessary after exporting final outputs. 
```

Comment:
You later get to copy and paste, but I think we already here should differentiate between different types of manual edits. Manual edits that reuire copy and paste is an absolute no-no with extremely few if any exceptions. But manual formatting is something that really should be avoided but is not as risky and copy-paste.

Reply:
addressed in ab0718b

Reply:
I like it, just made one edit in 1d1d14a58dda41170d1865d8df6f2b6a645d2918. I changed _risk-prone_ to _error-prone_. @luizaandrade , feel free to resolve this comment if you are ok with that edit